File: tests/input/demo/prime.py
def isPrime(x):
    if x < 2:
        return False

    if x == 2:
        return True

    d = 2

    while d * d <= x:
        if x % d == 0:
            return False
        d = d + 1

    return True


def calcPrimes(n):
    i = 2
    c = 0

    while c <= n:
        if isPrime(i):
            print(i)
            c = c + 1
        i = i + 1


calcPrimes(1000000)

File: tests/input/demo/prime.c
#include <stdio.h>
#include <stdbool.h>

bool isPrime(int x) {
    if (x < 2) {
        return false;
    }

    if (x == 2) {
        return true;
    }

    int d = 2;

    while (d * d <= x) {
        if (x % d == 0) {
            return false;
        }
        d = d + 1;
    }

    return true;
}

void calcPrimes(int n) {
    int i = 2;
    int c = 0;

    while (c <= n) {
        if (isPrime(i)) {
            printf("%d\n", i);
            c++;
        }
        i = i + 1;
    }
}

int main() {
    calcPrimes(1000000);
    return 0;
}